src/test/resources/features/Returns.feature

```diff
@@ -162,12 +162,42 @@ Feature: Submit ECL Return
     Then I should be see the amount of ECL need to pay <Expected content>
     Examples:
       | UK Revenue | APDays | AMLDays | Expected content                                                           |
-      | 7000000    | 314    | 92      | The amount of levy you need to pay for this financial year is £0.          |
+      | 7000000    | 314    | 92      | The amount of levy you need to pay for this financial year is £0.00.          |
       | 10000000   | 113    | 198     | The amount of levy you need to pay for this financial year is £5,424.65.   |
       | 31000000   | 284    | 300     | The amount of levy you need to pay for this financial year is £29,589.04.  |
       | 350000000  | 91     | 256     | The amount of levy you need to pay for this financial year is £175,342.46. |
 
   Scenario: User wants to go to check your answers page directly without providing any of the previous pages details
     Given I am signed in to the return journey
     When I go to the return submission check your answers page directly without providing answers for any of the previous page questions
-    Then I should be on the page that says The answers you provided are not valid
\ No newline at end of file
+    Then I should be on the page that says The answers you provided are not valid
+
+  Scenario: User wants to change their ECL contact details before submitting the returns
+    Given I am signed in to the return journey
+    When I click on the change link to modify my contact details
+    Then I should be on the page that says Check your answers
+
+  Scenario: User wants to change their ECL accounting period before submitting the returns
+    Given I am signed in to the return journey
+    When I click on the change link to modify my economic crime levy accounting period
+    Then I should be on the page that says Check your answers
```

Comment:
Changing anything pertaining to accounting period, revenue or AML activity will take you back to the amount due page, not the check your answers page. Once 'Save and continue' is clicked from the amount due page then you will either go to the contact name page (if you haven't already captured contact details) or the check your answers page (if contact details have already been captured).